From 2deedf339740abf0abd25eee3e98211cf5c7ec5a Mon Sep 17 00:00:00 2001
From: Max Horn <max@quendi.de>
Date: Tue, 13 Sep 2022 16:09:19 +0200
Subject: [PATCH 2/4] Don't invoke jl_init()

This is not needed when running this code from within Julia
---
 src/julia_gc.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/julia_gc.c b/src/julia_gc.c
index db02b5d39..c5c536467 100644
--- a/src/julia_gc.c
+++ b/src/julia_gc.c
@@ -759,7 +759,6 @@ void GAP_InitJuliaMemoryInterface(jl_module_t *   module,
     // that we can track objects allocated during `jl_init()`.
     max_pool_obj_size = jl_gc_max_internal_obj_size();
     jl_gc_enable_conservative_gc_support();
-    jl_init();
 
     SetJuliaTLS();
 #ifdef SKIP_GUARD_PAGES
-- 
2.38.1

